### PR TITLE
Add some RBAC docs

### PR DESCRIPTION
**Pull request by eyalb181**

### CODE REVIEW

File: content/docs/teams/security/index.md [hunk 1]
```diff
@@ -23,4 +23,56 @@ toc: true
     - `CAP_NET_ADMIN` - for modifying routing tables
     - `CAP_SYS_PTRACE` - for reading the target pod's environment variables
     - `CAP_SYS_ADMIN` - for joining the target pod's network namespace
-- Missing anything? Feel free to ask us on Discord or hi@metalbear.co
\ No newline at end of file
+- Missing anything? Feel free to ask us on Discord or hi@metalbear.co
+
+## How do I configure Role Based Access Control for mirrord for Teams?
+
+mirrord for Teams includes a built-in ClusterRole called `mirrord-operator-user`, which controls access to the Operator API. To grant access to the Operator API, you can create a ClusterRoleBinding like this:
```
Comment (DmitryDodzin) [inline content/docs/teams/security/index.md:30]: mirrord for Teams works on top of Kubernetes's built-in RBAC with two main resources `mirrordoperators` and `targets` and two sub-resources `mirrordoperators/certificate` and `targets/port-locks` under the `operator.metalbear.co` apiGroup.

By limiting the `targets` resource to specific namespaces you can limit the access to our Operator API.

** NOTE `mirrordoperators` is a cluster-scoped (not namespaced) therefore must be added via ClusterRole.

Comment (aviramha) [reply content/docs/teams/security/index.md:30]: We also need to mention that the cluster role is needed for `mirrord operator status` while the namespaced is only for `exec`

Comment (DmitryDodzin) [reply content/docs/teams/security/index.md:30]: We also perform status request before `exec`

Comment (aviramha) [reply content/docs/teams/security/index.md:30]: So you have to have cluster role.. no?

Comment (DmitryDodzin) [reply content/docs/teams/security/index.md]: At least one for mirrordstatus and it's subresource

File: content/docs/teams/security/index.md [hunk 2]
```diff
@@ -23,4 +23,60 @@ toc: true
...
     - `CAP_SYS_ADMIN` - for joining the target pod's network namespace
-- Missing anything? Feel free to ask us on Discord or hi@metalbear.co
\ No newline at end of file
+- Missing anything? Feel free to ask us on Discord or hi@metalbear.co
+
+## How do I configure Role Based Access Control for mirrord for Teams?
+
+mirrord for Teams works on top of Kubernetes' built-in RBAC with two main resources, `mirrordoperators` and `targets`,  and two sub-resources, `mirrordoperators/certificate` and `targets/port-locks` under the `operator.metalbear.co` apiGroup.
+
+By limiting the `targets` resource to specific namespaces you can limit the access the Operator has. The specific verbs for rules to our resources can be copied from the examples below.
```
Comment (DmitryDodzin) [inline content/docs/teams/security/index.md:32]:
```suggestion
By limiting the `targets` resource to specific namespaces you can limit the access to the Operator. The specific verbs for rules to our resources can be copied from the examples below.
```

**Pull request closed without merging**